openmetadata-ui/src/main/resources/ui/src/components/MyData/MyData.component.tsx

```diff
@@ -160,7 +160,7 @@ const MyData: React.FC<MyDataProps> = ({
                   data-testid="following-data"
                   to={getLinkByFilter(Ownership.FOLLOWERS)}>
                   <span className="link-text tw-font-normal tw-text-xs">
-                    View All
+                    View All <span>({followedData.length})</span>
```

Comment:
@ShaileshParmar11 `followedData.length` will not give us the total count. It will only have up to a maximum of 8 results since the size we request is 8.
<img width="771" alt="Screen Shot 2022-05-02 at 10 20 20 AM" src="https://user-images.githubusercontent.com/1051198/166300433-8528ac48-a32a-4465-9198-f96e49ea048b.png">

Reply:
@ShaileshParmar11 are you using the followedData.hits? that will give the right result

Reply:
Yes, now I am using the total value from hits!